### ISSUE TITLE
Apt_Repository fails in Packer/Docker build

### ISSUE THREAD

**Issue by munkyjunky** [ansible/ansible-modules-core#5535]
##### Issue Type
- Bug Report

### Summary
When using Ansible with Packer & Docker the `apt_repository` module fails to fetch PPA information. 
> failed to fetch PPA information, error was: Connection failure: unknown error (_ssl.c:2831)

When using Ansible as a provisioner within Packer (Docker builder) the apt_repository step fails to add the PPA. When using the same step using Vagrant & a VirtualBox VM works, which suggests (to me) an issue with the Docker connection.

### Expected Behaviour
The PPA is added

### Actual Behaviour 
The PPA is not added

**Packer, Ansible config, packer output**
https://gist.github.com/munkyjunky/b7bc8666cb34d1ca28a8530511a3ca06

**Version**
Mac OS Sierra (host)
Ansible 2.1.2.0
Packer v0.11.0
Docker version (below)
```
Client:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.7.1
 Git commit:   6f9534c
 Built:        Thu Sep  8 10:31:18 2016
 OS/Arch:      darwin/amd64

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 17:52:38 2016
 OS/Arch:      linux/amd64
```

**Comment by ansibot** [comment]
@munkyjunky, we need more information to resolve this issue. For that reason, we put this Issue into the 'needs_info' state.

Here are the required items we could not find in your description:
- issue type

When you have filled in the missing data, we will notify the module maintainer for further action.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by munkyjunky** [comment]
After a bit of debugging I've discovered this fails because the command `add-apt-repository` is not present on the 16.04 docker image. By installing `software-properties-common` to the system first, the command works. 

I'm unsure why this worked for me using a VirtualBox VM - my only thought would be that the VM image I used already had the required packages installed. Looks like this isn't an issue with Ansible, although a more descriptive error message would have been helpful!
